doc/nrf/ug_thread_configuring.rst

```diff
@@ -136,3 +136,43 @@ By default, when a Thread device is configured as MTD, it operates as Minimal En
 You can choose to make it operate as Sleepy End Device (SED) by enabling the :option:`CONFIG_OPENTHREAD_MTD_SED` option.
 
 For more information, see `Device Types on OpenThread portal`_.
+
+UART recommendations for NCP
+****************************
+
+Use the following recommended default UART settings for configuration based on :ref:`thread_architectures_designs_cp_ncp` architecture:
+
+* Bit rate: 115200 (or 1000000)
```

Comment:
I would say that it mainly supports 1M baud rate and 115200 on the second place. Maybe it's even worth to write that 115200 baud rate is not recommended in serious use cases - only in some tests, because it is too slow to forward whole network traffic to co-processor and may affect in lowering throughput (E.g. in NCS NCP sample by default supports 1M baud rate).

Reply:
Edited to 1 milion.